tests/main.test.js

```diff
@@ -69,3 +77,15 @@ test('media queries print removed', async () => {
   const { finalCss } = await runMinimalcss('media-queries-print')
   expect(finalCss).toEqual(result)
 })
+
+test('handles 307 CSS file', async () => {
+  const result = 'p{color:violet}'
+  const { finalCss } = await runMinimalcss('307css')
+  expect(finalCss).toEqual(result)
+})
+
+// test('handles 307 HTML file', async () => {
```

Comment:
https://github.com/stereobooster/puppeteer-example/tree/307-issue it works as standalone version :/